src/Restrict/Native/Admin.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace FileUploadTypes\Restrict\Native;
+
+use FileUploadTypes\Allowed;
+
+/**
+ * Native file types.
+ *
+ * @since {VERSION}
+ */
+class Admin {
+
+	/**
+	 * Unflitered WP types.
+	 *
+	 * @since {VERSION}
+	 *
+	 * @var string[]
+	 */
+	private static $unfiltered_types;
+
+	/**
+	 * Allowed object.
+	 *
+	 * @since {VERSION}
+	 *
+	 * @var Allowed
+	 */
+	private $allowed;
+
+	/**
+	 * The constructor.
+	 *
+	 * @since {VERSION}
+	 *
+	 * @param Allowed $allowed Allowed object.
+	 */
+	public function __construct( Allowed $allowed ) {
+
+		$this->allowed = $allowed;
+	}
+
+	/**
+	 * Register hooks.
+	 *
+	 * @since {VERSION}
+	 *
+	 * @return void
+	 */
+	public function hooks() {
+
+		add_action( 'fileuploadtypes_settings_display_types_table_after_enabled_types', [ $this, 'table_rows_with_native_types' ] );
+		add_filter( 'fileuploadtypes_migrations_dispatcher_add_native_file_upload_types_callback', [ $this, 'dispatcher_get_migrations_list' ] );
+		add_filter( 'fileuploadtypes_settings_transform_native_type', [ $this, 'get_upload_type' ] );
+	}
+
+	// phpcs:disable Generic.Metrics.CyclomaticComplexity.TooHigh
+	/**
+	 * Display table rows with native types which are not enabled.
+	 *
+	 * @since {VERSION}
+	 *
+	 * @return void
+	 */
+	public function table_rows_with_native_types() {
+
+		$stored_types  = get_option( 'file_upload_types', [] );
+		$native_types  = isset( $stored_types['native'] ) ? (array) $stored_types['native'] : [];
+		$enabled_types = isset( $stored_types['enabled'] ) ? (array) $stored_types['enabled'] : [];
+
+		if ( ! empty( $native_types ) ) :
+
+		?>
+		<tr class="section">
```

Comment:
I can completely do this, but I vote to create an GH issue for this and handle separately as this is not 100% related to the task. I mean this piece of code is, but what do you think about introducing templates for all html we already have? This code to echo pieces of `<table>` is here and there: it would be great to convert them to templates as a separate task, what to you say?